ksql-engine/src/test/java/io/confluent/ksql/function/udf/datetime/StringToTimestampTest.java

```diff
@@ -84,7 +90,12 @@ public void shouldBeThreadSafe() {
     IntStream.range(0, 10_000)
         .parallel()
         .forEach(idx -> {
-          shouldCovertStringToTimestamp();
+          try {
+            shouldCovertStringToTimestamp();
+          } catch (ParseException e) {
+            e.printStackTrace();
```

Comment:
guess this is fine since it's just test code, but is this here for something other than debugging? seems like the ParseException should be enough to indicate something is wrong and then you could just try to reproduce locally under a debugger.

Reply:
perhaps fail with the message from the `ParseException`?

Reply:
Used the ParseException message for fail as @dguy suggested.